doc/source/whatsnew/v2.2.0.rst

```diff
@@ -415,6 +415,7 @@ MultiIndex
 
 I/O
 ^^^
+- Bug in :func:`DataFrame.describe` when formatting percentiles in the resulting percentile 99.999% is rounded to 100% (:issue:`55765`)
```

Comment:
Can you put this in the `Other` section?

Reply:
Sure. Done